spec/services/notifications/milestone/send_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+
+RSpec.describe Notifications::Milestone::Send do
```

Comment:
I would add a test (or tests) checking the notifications `json_data`

Reply:
I have added a test for json_data.

Reply:
:+1: I suggest checking a couple of more fields (like id or title), not only a class name.

Reply:
Added 2 more fields